sklearn/decomposition/tests/test_pca.py

```diff
@@ -684,5 +685,9 @@ def test_pca_svd_output(nfeat, seed):
     X_tranformed2 = U2[:, 0] * s2[0]
     X_tranformed2 = np.array([round(abs(v), valNum) for v in X_tranformed2])
 
-    assert np.testing.assert_array_almost_equal(X_tranformed, X_tranformed1, decimal=1)
-    assert np.testing.assert_array_almost_equal(X_tranformed, X_tranformed2, decimal=1)
+    assert np.testing.assert_array_almost_equal(
+        X_tranformed[:lenNum], X_tranformed1[:lenNum], decimal=1
+    )
+    assert np.testing.assert_array_almost_equal(
+        X_tranformed[:lenNum], X_tranformed2[:lenNum], decimal=1
+    )
```

Comment:
Hello, It’s been a while since I submitted pr, I wonder whether my pr is useful or will it be merged later.
Anyway, thank you for your help during this time.